src/taco-dependency-installer/test/androidSdkInstaller.ts

```diff
@@ -0,0 +1,131 @@
+/**
+ *******************************************************
+ *                                                     *
+ *   Copyright (C) Microsoft. All rights reserved.     *
+ *                                                     *
+ *******************************************************
+ */
+
+/// <reference path="../../typings/dependencyInstallerInterfaces.d.ts"/>
+/// <reference path="../../typings/mocha.d.ts"/>
+/// <reference path="../../typings/mockery.d.ts"/>
+/// <reference path="../../typings/should.d.ts"/>
+
+"use strict";
+
+/* tslint:disable:no-var-requires */
+// var require needed for should module to work correctly
+// Note not import: We don't want to refer to shouldModule, but we need the require to occur since it modifies the prototype of Object.
+var shouldModule = require("should");
+/* tslint:enable:no-var-requires */
+
+import installerProtocol = require("../elevatedInstallerProtocol");
+import ILogger = installerProtocol.ILogger;
+import mockery = require("mockery");
+import Q = require("q");
+import tacoTestsUtils = require("taco-tests-utils");
+import tacoUtils = require("taco-utils");
+import _ = require("lodash");
+
+import nodeFakes = tacoTestsUtils.NodeFakes;
+
+type TelemetryEvent = TacoUtility.ICommandTelemetryProperties;
+
+class FakeLogger implements ILogger {
+    public log(message: string): void {
+        // TODO: Store all messages in memory, so we can test them
```

Comment:
If we really plan to test the messages, we should add an issue number here. If not, I would replace the "TODO" comments with "/* empty */"